### ISSUE TITLE
[Docs] Some missing information about soft_teacher

### ISSUE THREAD

**Issue by RangeKing** [open-mmlab/mmdetection#9184]
### Branch

3.x branch https://mmdetection.readthedocs.io/en/3.x/

### 📚 The doc issue

1. Metafile is missing in https://github.com/open-mmlab/mmdetection/tree/3.x/configs/soft_teacher
2. I saw the performance table in https://github.com/open-mmlab/mmdetection/blob/3.x/docs/en/user_guides/semi_det.md#configure-semi-supervised-model, but not in https://github.com/open-mmlab/mmdetection/blob/3.x/configs/soft_teacher/README.md

### Suggest a potential alternative/fix

Refine the docs and metafile in https://github.com/open-mmlab/mmdetection/tree/3.x/configs/soft_teacher.

**Comment by Czm369** [comment]
Thanks for your suggestions. Due to the low accuracy of semi-supervised training models based on partial coco training sets, they are generally not used for downstream tasks, so the relevant models are not released. In addition, due to the large number of semi-supervised hyperparameters, we are still experimenting and will release some high-precision models in the future.

**Comment by austinmw** [comment]
@Czm369 Hi, do you know if MMDet's implementation achieves the same accuracy as in the paper? I'm currently seeing lower mAP than reported on coco 1% k-fold 1, so wanted to double check, while I continue trying to debug.

**Comment by Czm369** [comment]
The accuracy of my experiment is aligned with the paper.
